### PR TITLE
OCPBUGS-30827: Adding SDN deprecation statements across the doc set

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.15+

Issue:
https://issues.redhat.com/browse/OCPBUGS-30827

Link to docs preview:
https://73316--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/about-networking

This deprecation language is already reviewed and approved by PM, as it was included in the release notes. This PR adds the language to the body of the text in the form of a snippet.

### CODE REVIEW

**Comment by jab-rh** [comment]
@ahardin-rh, I wonder about the feature matrix: https://docs.openshift.com/container-platform/4.15/networking/openshift_sdn/about-openshift-sdn.html#nw-ovn-kubernetes-matrix_about-openshift-sdn

This module repeats in the OVN-Kubernetes about as well. This probably just needs to list the features for OVN-Kubernetes and can be therefore simplified. For 4.17 it would definitely come out. Not sure for 4.16.

File: snippets/sdn-deprecation-statement.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Text snippet included in the following assemblies:
+//
+// * networking/about-networking.adoc
```
Comment (dfitzmau) [inline snippets/sdn-deprecation-statement.adoc:3]: I was unable to open the following assemblies: 
Screenshot from 2024-04-05 16-15-56
Screenshot from 2024-04-05 16-17-23

File: networking/openshift_sdn/migrate-to-openshift-sdn.adoc [hunk 2]
```diff
@@ -8,6 +8,8 @@ toc::[]
 
 As a cluster administrator, you can migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
```
Comment (dfitzmau) [inline networking/openshift_sdn/migrate-to-openshift-sdn.adoc:9]: This file does not render in the preview. Not sure why.

File: networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc [hunk 3]
```diff
@@ -8,6 +8,8 @@ toc::[]
 
 As a cluster administrator, you can rollback to the OVN-Kubernetes network plugin from the OpenShift SDN network plugin if the migration to OpenShift SDN is unsuccessful.
```
Comment (dfitzmau) [inline networking/openshift_sdn/rollback-to-ovn-kubernetes.adoc:9]: This file does not render in the preview.

**Comment by dfitzmau** [comment]
Hi @ahardin-rh . Overall LTGM, but I cannot access two files on the preivew. Am I missing something?

File: snippets/sdn-deprecation-statement.adoc (hunk 1 above)
Comment (ahardin-rh) [reply snippets/sdn-deprecation-statement.adoc:3]: https://73316--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/about-networking seems to render fine on my end

File: networking/openshift_sdn/migrate-to-openshift-sdn.adoc (hunk 2 above)
Comment (ahardin-rh) [reply networking/openshift_sdn/migrate-to-openshift-sdn.adoc:9]: @dfitzmau Thank you! I will re-check these two files.

**Comment by ahardin-rh** [comment]
No QE needed since the language itself is already approved. Looks like some files were removed in 4.15, which is why they don't render in preview. I think we can merge as it is.

**Pull request merged**